data/card_catalogue/unique-1.json

```diff
@@ -379,20 +249,10 @@
       "image": "assets/core/card/illustration/Tripwire Charm.png"
     }
   },
-  {
-    "title": "Counterspell",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "The effect of the next spell intended for you is applied instead to the caster.",
```

Comment:
We don't have an official notion of a 'spell'. Given that, the effect of this card is too niche

Reply:
I agree with the comment and the removal but will miss the card, maybe there's something for a future card along broader terms like "the next debuff"

Reply:
Once we have a notion of debuffing someone outside of just simple damage then I think we'll remake this card naturally